spinn_front_end_common/interface/config_handler.py

```diff
@@ -55,6 +55,12 @@ class ConfigHandler(object):
         #
         "_provenance_file_path",
```

Comment:
If the idea is to split the provenance in two why is the original provenance folder still there?

Reply:
becuase there are prov items which are not graph related. aka pacman, router, etc etc